srcpkgs/NoiseTorch/template

```diff
@@ -1,17 +1,29 @@
 # Template file for 'NoiseTorch'
 pkgname=NoiseTorch
-version=0.11.4
+version=0.12.2
 revision=1
 build_style=go
 go_ldflags="-X main.version=${version} -X main.distribution=VoidLinux"
 go_import_path="noisetorch"
 depends="libcap-progs"
 short_desc="Real-time microphone noise suppression on Linux"
-maintainer="Soc Virnyl S. Estela <renegan.ronin@gmail.com>"
+maintainer="yosh <yosh-git@riseup.net>"
```

Comment:
I would prefer if you put in `Orphaned <orphan@voidlinux.org>` and only adopt the package if you update or review it again. We have too many people adopting packages they never update again and when reviewing updates we have to chase the maintainer field to see if they are still active or not.

Reply:
Noted. I don't actively use this software, rather this was made purely to address the issues, so I will most likely not update or review it again.